feed_remote.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+set -e
+
+# Remote feed script that works against the Kubernetes cluster
+# Uses kubectl to run bash scripts inside the cluster
+
+NAMESPACE=${NAMESPACE:-fold}
+
+echo "Running feed sequence against remote Kubernetes cluster..."
+
+# Helper function to run bash scripts in the cluster
+run_script() {
```

Comment:
Don't do it this way. Look at the Codebase. The cycle starts when something lands in the work queue as a seed. You don't have to call any of the rust binaries for this to work.